War on typos Act 1

szepeviktor opened the pull request:
- [x] The contributed code is licensed under GPL v2.0
- [x] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran - Is there generated code in the repository ❔
- [ ] Documention updated
- [ ] Test suite update

Hello!
Fixed misspellings in `bin/` with a tool called `typos`.
There may be collateral damage. Please review.

After this one gets merged I proceed with other directories 🚀

it-percona-cla commented:
[![CLA assistant check](https://cla-assistant.percona.com:443/pull/badge/signed)](https://cla-assistant.percona.com:443/percona/percona-toolkit?pullRequest=655) <br/>All committers have signed the CLA.

szepeviktor commented:
🎈 Fun fact: "arugment" is the TOP typo.

szepeviktor commented:
At the end `typos` could run in a CI workflow!

szepeviktor commented:
@svetasmirnova Friendly ping 🏓

svetasmirnova reviewed (changes requested):
You changed only tools code while Percona Toolkit auto-generates most of the tool code from modules. As a result, all your work will be wiped after we run the update-modules script. Please take a look at Contributing guide for details. Please move your changes to the proper places.

szepeviktor commented:
Thank you Света!
I thought the opposite, left lib/* files alone :)

Fixing ... 🏃🏻‍♂️

szepeviktor committed:
Fix typos in lib/ directory

szepeviktor committed:
Update generated bin/ files

szepeviktor commented:
@svetasmirnova Done 🍏
I've put typo fixes and generated files in separate commits for you.

szepeviktor commented on lib/RowChecksum.pm:
```diff
@@ -169,7 +169,7 @@ sub make_chunk_checksum {
 
    my %crc_args = $self->get_crc_args(%args);
-   PTDEBUG && _d("Checksum strat:", Dumper(\%crc_args));
+   PTDEBUG && _d('Checksum start:', Dumper(\%crc_args));
```
I am unsure here.

svetasmirnova replied:
Should be OK

svetasmirnova committed:
PR 655 -  War on typos Act 1 #655

- Updated modules in tools that were not updated
- Fixed tests to reflect proposed changes

svetasmirnova commented:
Thank you! I added more changes, so the tests are now passed.

szepeviktor commented:
Thank you for your commit!

I was looking for an `update-everything-everywhere.sh` script but could not find it.
Contributors would find it useful.
